### PR TITLE
Rewrite release (closes #585)

### LINKED ISSUE #585: release.js needs to be aware of new .map files
When releasing, the release.js tool is not yet aware of the .map files being generated now by Require.js

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
This is a pretty critical patch. Can I kindly ask maintainers to review it ASAP.

File: tools/build-w3c-common.js [hunk 1]
```diff
@@ -18,18 +18,24 @@ colors.setTheme({
   warn: "yellow",
 });
 
-const buildW3C = async(function*(aVersion) {
-  aVersion = (!aVersion) ? "latest" : aVersion;
+const buildW3C = async(function*() {
+  var versionsToBuild = Array.from(arguments);
```
Comment (marcoscaceres) [inline tools/build-w3c-common.js]: should be `let`.

**Comment by halindrome** [comment]
Not sure why the build test failed, but the general changes look okay.  I want to use release.js in anger, but that is not critical path.

**Comment by marcoscaceres** [comment]
Taking that as an r+ :+1:

**Pull request merged**